### PR TITLE
Add json output option to komodo-non-deployed

### LINKED ISSUE #301: Add json output from komodo-non-deployed
For integration with Github action, it is relevant to be able to output in json format from `komodo-non-deployed`, in order to avoid the hackery in 
https://github.com/equinor/komodo-releases/pull/3043/files#diff-487281c14a5ff9f976920b3406c32dea2972482cce2375fa3d53ffe38adefcefR50

### PR DESCRIPTION
**Pull request by berland**
This eases integration with Github actions

Resolves #301

### CODE REVIEW

File: komodo/deployed.py [hunk 1]
```diff
@@ -62,6 +72,7 @@ def deployed_main():
         default=None,
         help="The maximum number of undeployed matrices to list.",
     )
+    parser.add_argument("--json", action="store_true", help="Get output in JSON format")
```
Comment (kwinkunks) [inline komodo/deployed.py:75]: Should perhaps add explicit default to CLI, and mention default in Help.

Comment (berland) [reply komodo/deployed.py:76]: The default is automatically set to `False`, but needed to add a `formatter_class` to get the default visible. Fixed.

**Review by kwinkunks — APPROVED** [review]
This looks good to me, assuming it provides the output you need in the GitHub workflow it services (I didn't check this)

**Comment by berland** [comment]
> This looks good to me, assuming it provides the output you need in the GitHub workflow it services (I didn't check this)

Apparently me neither.. https://github.com/equinor/komodo/pull/311

**Pull request merged**